CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 - Minor: Include number of quest completions and quest points in quest notifications. (#178)
 - Minor: Collapse all notifier configuration sections by default. (#176)
 - Minor: Include number of completed entries in collection log notification. (#174, #181)
-- Bugfix: Include PKer from Last Man Standing minigame in death notifications. (#187)
+- Bugfix: Avoid death notifications for more safe activities. (#194)
```

Comment:
Can we change this to a Minor changelog entry, and list which areas have changed?
CC @Felanbird for feedback on this

Reply:
ya sure